charts/stable/sharry/values.yaml

```diff
@@ -1,77 +1,40 @@
-# Default values for sharry.
-
+#
+## IMPORTANT NOTE
+##
+## This chart inherits from our common library chart. You can check the default values/options here:
+## https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+##
 image:
-  # -- sharry image
   repository: eikek0/sharry
-  # -- sharry image tag
   tag: 1.6.0
-  # sharry image pull policy
   pullPolicy: IfNotPresent
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #  cpu: 100m
-  #  memory: 128Mi
-  # requests:
-  #  cpu: 100m
-  #  memory: 128Mi
-
 service:
   port:
-    # -- Kubernetes port where the GUI is exposed
     port: 9090
 
 args:
 - "/opt/sharry.conf"
 
-sharry:
-  # -- This is the base URL this application is deployed to.
-  baseUrl: "http://localhost:9090"
-  # -- By default a H2 file-based database is configured. You can provide a postgresql or mariadb connection here.
-  jdbcUrl: "jdbc:h2://\"${java.io.tmpdir}\"/sharry-demo.db;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE"
-  # -- jdbc Username
-  jdbcUser: sa
-  # -- jdbc Password
-  jdbcPassword: ""
-  # -- When storing binary data use chunks of this size.
-  chunkSize: "512K"
-  # -- Maximum size of a share.
-  maxSize: "1.5G"
-  # -- Maximum validity for uploads.
-  maxValidity: "365 days"
-  # -- The mode defines if new users can signup or not. It can have three values : open, invite, closed
-  signup: "open"
-
-# -- Node labels for pod assignment of the GUI
-nodeSelector: {}
-
-# -- Toleration labels for pod assignment of the GUI
-tolerations: []
-
-# -- Affinity settings for pod assignment of the GUI
-affinity: {}
-
-# -- Pod annotations
-podAnnotations: {}
-
-additionalVolumes:
-- name: sharry-config
-  configMap:
-    name: sharry-config
-additionalVolumeMounts:
-- name: sharry-config
-  mountPath: /opt/sharry.conf
-  subPath: sharry.conf
+# -- Full list of options https://eikek.github.io/sharry/doc/configure
+# @default -- see URL to default config
+config: |
+ sharry.restserver {
+    backend {
+      auth {
+        fixed {
+          enabled = false  # set to true to enable this auth provider
+          user = "admin"
+          password = "admin"
+          order = 10
+        }
+      }
+    }
+  }
 
 ingress:
   enabled: false
   annotations:
-    # -- Nginx client Body Buffer Size
     nginx.ingress.kubernetes.io/client-body-buffer-size: "2048m"
-    # -- Nginx Proxy Body Size
     nginx.ingress.kubernetes.io/proxy-body-size: "2048m"
     nginx.ingress.kubernetes.io/proxy-buffering: "off"
```

Comment:
We should probably comment these out, not everyone is using nginx

Reply:
Indeed